earth_enterprise/src/SConstruct

```diff
@@ -367,6 +367,37 @@ installdirs = {
     'htdocs': InstServerDir('gehttpd/htdocs'),
 }
 
+def get_cc_version(cc_path):
+  """Extracts the version number of a C/C++ compiler (like GCC) as a list of
+  strings."""
+
+  process = subprocess.Popen(
```

Comment:
Minor: I know this is @tst-ppenev's code, but you could simplify this by using the check_output function.  It returns the output directly so you don't have to deal with redirecting the output and using process.communicate().

Reply:
I don't think that's available in Python 2.6 on RHEL 6.

Reply:
sorry, that looks like out of scope for this ticket.

Reply:
I'm OK with leaving it as is.